components/page-header/page-header.component.ts

```diff
@@ -45,11 +47,18 @@ const NZ_CONFIG_MODULE_NAME: NzConfigKey = 'pageHeader';
         <!--back-->
         @if (nzBackIcon !== null) {
           <div (click)="onBack()" class="ant-page-header-back">
-            <div role="button" tabindex="0" class="ant-page-header-back-button">
+            <button
+              nz-button
```

Comment:
As far as I know, in antd4, the back button is a div instead of a button.

https://4x-ant-design.antgroup.com/components/page-header-cn/

Reply:
That's true, but i tought we could have a button because, we have to explicity set the role of the div as button. Button is more appropriate in this case no ? 

Anyway if you think it's better to have let's have a div, and i will rework a little bit this PR to have the correct style

Reply:
While I also think there should be a button here, since we need to synchronize with antd, it can only be a div for now :)

Reply:
That's okey let me rework so :)

Reply:
@HyperLife1119 done :)